python/create_client/create_client.py

```diff
@@ -277,9 +286,9 @@ def _do_websocketserver_handshake(self):
         response = self._shotgun_connection._call_rpc(
             "retrieve_ws_server_secret", {"ws_server_id": self._server_id}
         )
-        ws_server_secret = response["ws_server_secret"]
-        if ws_server_secret[-1] != "=":
-            ws_server_secret += "="
+        ws_server_secret = six.ensure_binary(response["ws_server_secret"])
+        if ws_server_secret[-1:] != b"=":
```

Comment:
asking for an index of a binary array will return a int, so we use a slice instead to get the actual characters